### PR TITLE
Development docs & Miscellaneous API addition

### PR DESCRIPTION
**Pull request by Thundermaker300**
* Doc stuff: as always
* Moving Player::GetCameraById to (static) Map::GetCameraById (having it as a non-static in the player class literally made no sense)
* Adding an overload to Player::SetCamera to take a camera object

### CODE REVIEW

**Review by PintTheDragon — CHANGES_REQUESTED** [review]
This should be added back with a deprecation notice. Any plugins using it will break.

**Comment by PintTheDragon** [comment]
Wait I think I did it wrong fuck. It's the line where you remove the function from Player.

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -925,28 +937,24 @@ public static Player Get(string args)
             }
         }
 
-        /// <summary>
```
Comment (PintTheDragon) [inline Exiled.API/Features/Player.cs:928]: This should be added back with a deprecation notice. Any plugins using it will break.

**Comment by PintTheDragon** [comment]
I think I did it right this time.

**Comment by PintTheDragon** [comment]
Wait I'm blind lmao you already did it

**Review by PintTheDragon — APPROVED** [review]
Yeah it's good

**Comment by Thundermaker300** [comment]
Thanks, Pint

**Review by iRebbok — APPROVED** [review]

**Pull request merged**